packages/editor/src/client/panel/peerEditPanel.tsx

```diff
@@ -190,27 +191,42 @@ export function PeerEditPanel({id, onClose, onSave}: ImageEditPanelProps) {
           </Form>
         </Panel>
         {!isLoadingPeerProfile && isValidURL && (
-          <Panel header={t('peerList.panels.information')}>
-            <Panel
-              bordered={true}
-              style={{
-                height: '200px',
-                marginBottom: '20px',
-                backgroundSize: 'cover',
-                backgroundImage: `url(${
-                  profile?.logo?.previewURL ?? 'https://via.placeholder.com/240x240'
-                }`
-              }}
-            />
-            <DescriptionList>
-              <DescriptionListItem label={t('peerList.panels.name')}>
-                {profile?.name}
-              </DescriptionListItem>
-              <DescriptionListItem label={t('peerList.panels.themeColor')}>
-                {profile?.themeColor}
-              </DescriptionListItem>
-            </DescriptionList>
-          </Panel>
+          <>
```

Comment:
Is this <> fragment is needed? Else this whole block wouldn't show up in the diff.